File: Codewars/8kyu/8kyu_return_negative/src/main.go
package main

import (
	"fmt"
)

func MakeNegative(num int) int {
	if 0 >= num {
		return num
	}
	return -num
}

func main() {
	fmt.Println(-1 == MakeNegative(1))
	fmt.Println(-5 == MakeNegative(-5))
	fmt.Println(0 == MakeNegative(0))
}

File: Codewars/8kyu/8kyu_return_negative/src/main.js
'use strict'

const assert = require('assert')

function mysolution(num) {
    return 0 > num? num: -num
}

function othersolution(num) {
    return -Math.abs(num)
}

const makeNegative = othersolution

describe('test', () => {
    it('1', () => {
       assert.equal(makeNegative(42), -42) 
    }),
    it('2', () => {
       assert.equal(makeNegative(-5), -5) 
    }),
    it('3', () => {
        assert.equal(makeNegative(0), 0)
    })

})